Neos.FluidAdaptor/Classes/Core/Cache/CacheAdaptor.php

```diff
@@ -68,7 +68,7 @@ public function flush($name = null)
         if ($name !== null) {
             return $this->flowCache->remove($name);
         } else {
-            return $this->flowCache->flush();
+            $this->flowCache->flush();
```

Comment:
I don't get, why we should remove the return here. But if we need to, we should remove both.

Reply:
PhpStan doesnt like to return `void` when returned from another method. Will adjust to add a `return null` below